contract/src/lib.rs

```diff
@@ -5,6 +5,10 @@ use std::collections::{HashMap, HashSet};
 
 type ParticipantId = u32;
 
+pub mod hpke {
+    pub type PublicKey = [u8; 32];
```

Comment:
Yeah, it was a whole ordeal to make use of the `keys` crate to provide this one simple type. So I just hardcoded it for now since it wasn't worth the effort to put it inside the crate. There's probably some artifacts left from that